crates/store/src/db/tests.rs

```diff
@@ -95,6 +95,32 @@ fn test_sql_insert_nullifiers_for_block() {
     }
 }
 
+#[test]
+fn test_sql_insert_transactions() {
+    let mut conn = create_db();
+
+    let count = insert_transactions(&mut conn);
+
+    assert_eq!(count, 2, "Two elements must have been inserted");
+}
+
+#[test]
+fn test_sql_select_transactions() {
+    fn query_transactions(conn: &mut Connection) -> Vec<RpoDigest> {
+        sql::select_transactions_by_accounts_and_block_range(conn, 1, 2, &[1]).unwrap()
+    }
+
+    let mut conn = create_db();
+
+    let transactions = query_transactions(&mut conn);
+
+    assert!(transactions.is_empty(), "No elements must be initially in the DB");
+
+    let count = insert_transactions(&mut conn);
+
+    assert_eq!(count, 2, "Two elements must have been inserted");
+}
```

Comment:
can we also query after inserting the transactions and make sure it's also 2?

Reply:
Good catch, thank you!